packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -390,6 +423,40 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
       return Promise.resolve(undefined);
```

Comment:
This async function already contains `await`. Remove all `Promise.resolve`/`Promise.reject` and just return result/throw error.

Reply:
Fixed.